addons/account_edi/views/account_move_views.xml

```diff
@@ -39,8 +39,7 @@
                 </xpath>
                 <xpath expr="//div[@name='journal_div']" position="after">
                     <field name="edi_document_ids" invisible="1" />
-                    <field name="edi_state" invisible="1" />
-                    <field name="edi_state" attrs="{'invisible': ['|', ('edi_document_ids', '=', []), ('state', '=', 'draft')]}"/>
+                    <field name="edi_state" attrs="{'invisible': ['|', '|', ('edi_state', '=', False), ('edi_document_ids', '=', []), ('state', '=', 'draft')]}"/>
```

Comment:
Why is this edi_document_ids needed?  I would think that edi_state is False in that case or do we need a depends on edi_documents itself then? @bfr-o

Reply:
Indeed